### ISSUE TITLE
Duplicate output lines in Xcode console

### ISSUE THREAD

**Issue by mcicak** [lengocduy/swift-log#13]
**Describe the bug**
A single log statement results in two output log lines in Xcode console

**To Reproduce**
Initialize logger in AppDelegate:
```
LoggerManager.sharedInstance.initialize()
```
Make log statement like:
```
Log.error(message: "Some error")
```
Xcode console output will contain:
```
2021-10-24T21:34:46.406+0200 [❗️][MyClass.swift:36:main()] -> Some error
"2021-10-24T21:34:46.407+0200 [❗️][MyClass.swift:36:main()] -> Some error"
```
Notice the second line has "..." quotes.

**Expected behavior**
A single log statement should result in single console output line.

**iOS Simulator:**
 - OS: iOS 15.0

**Comment by lengocduy** [comment]
Thanks for report @mcicak . 
Will look into it, need to get iOS 15.0

**Comment by lengocduy** [comment]
Hi @mcicak , 
As default I enable all of these built-in loggers: `Console, DebugConsole, and File`.
The `"..."` quotes were for DebugConsole.

If you want to customize for console only, You can use as below
```
let logFormatter = LogFormatterImpl()
LoggerManager.sharedInstance.addLogging(LoggerFactoryImpl.makeConsoleLogging(logFormatter: logFormatter))
#if !DEBUG

// Disable LogLevels in release. Enable all LogLevels by default
LoggerManager.sharedInstance.disableLogLevels(LogLevel.allCases)
#endif
```

Can you please check and let me know if you have any issues?

**Comment by lengocduy** [comment]
Hi @mcicak,
Released Version `v1.2.1` to fix the issue, 
Just re-open this issue if you still see double lines, Thanks

**Comment by mcicak** [comment]
Indeed the issue is solved after pod update. Thank you.
